CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ add_subdirectory(SegmentEditorMaskVolume)
 add_subdirectory(SegmentEditorSurfaceCut)
 add_subdirectory(SegmentEditorWatershed)
 add_subdirectory(SegmentEditorDrawTube)
+add_subdirectory(SegmentEditorCropSegment)

```

Comment:
We chose CropSegment to be consistent with MaskSegment. I am definitely against Crop VOlume which does not explain sufficiently, plus will confuse the user with all other Crop Volume options. If you feel like CropToSegment is a better description, feel free to rename.

Reply:
Masking effect name is "Mask volume", not "Mask segment". Anyway, I agree that it is better to avoid having the same name for a module. Since the main differentiator compared to Mask volume effect that it operates on many segments at once (otherwise we could have just added a cropping checkbox to Mask volume effect), we could name it as "Split volume". "Crop to segment" would be appropriate if we could crop different kinds of nodes, not just volumes.